feat(config): init for gdzw7-eco

zackbcom opened the pull request:
## Changes
- Added config for [
GDZW7-ECO - Ecolink 700 Series Garage Door Controller ](https://products.z-wavealliance.org/products/4475)
- Params from debug and product manual. 
- [gdzw7-eco_debug.json](https://github.com/zwave-js/node-zwave-js/files/13781675/gdzw7-eco_debug.json)

zackbcom commented:
@AlCalzone What is the best way to test configs on the ui?

I'm running [zwave-js-ui](https://github.com/zwave-js/zwave-js-ui) on a docker container. Am I able to copy the configs into the container and re-integrate the device to test these changes? Do you know the location if that is the easiest way?

Thanks.

zackbcom commented:
@AlCalzone ready for review

AlCalzone commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+		},
+		{
+			"#": "2",
+			"label": "Relay Switch Period",
+			"description": "Number of milliseconds to turn on relay switch for.",
```
Dito
```suggestion
			"label": "Relay Turn On Duration",
```

AlCalzone commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+			"#": "5",
+			"$import": "~/templates/master_template.json#base_1-100_nounit",
+			"label": "Accelerometer Sensitivity",
+			"description": "Accelerometer vibration detection sensitivity: 1 is least sensitive, 100 is most sensitive.",
```
```suggestion
			"description": "1 is least sensitive, 100 is most sensitive.",
```

AlCalzone commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+			"$import": "~/templates/master_template.json#base_0-10_nounit",
+			"label": "Application-Level Retries",
+			"description": "Number of attempts on top of the stack-level retries to try to reach the controller/hub with Z-Wave messages.",
+			"unit": "attempts",
```
Unit can be removed too.

zackbcom replied:
removed `unit`

AlCalzone commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+			"maxValue": 5000,
+			"defaultValue": 1000 // firmware had default: 2000
+		},
+		{
+			"#": "3",
+			"$import": "~/0x014a/templates/ecolink-template.json#ecolink_5-60_seconds",
+			"label": "Door Open Timeout",
+			"description": "Seconds allowed for garage door to open until timeout.",
+			"defaultValue": 20 // firmware had default: 30
+		},
+		{
+			"#": "4",
+			"$import": "~/0x014a/templates/ecolink-template.json#ecolink_5-60_seconds",
+			"label": "Door Close Timeout",
+			"description": "Seconds allowed for garage door to close until timeout.",
+			"defaultValue": 15 // firmware had default: 30
+		},
```
Both descriptions are unnecessary.

zackbcom replied:
removed `descriptions`

AlCalzone reviewed (changes requested):
Sorry for the delay - I just got back from parental leave. I'd have fixed this myself, but the comments about the defaults left me confused...
What are the default values the device has when freshly reset and paired? The ones in the JSON file or the ones in the comments behind them?

Other than that I have a few minor remarks:

zackbcom commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
```
removed `associations`

zackbcom commented on packages/config/config/devices/0x014a/gdzw7-eco.json:
```diff
@@ -0,0 +1,79 @@
+			"#": "6",
+			"$import": "~/templates/master_template.json#base_0-10_nounit",
+			"label": "Application-Level Retries",
+			"description": "Number of attempts on top of the stack-level retries to try to reach the controller/hub with Z-Wave messages.",
```
removed `description`

zackbcom commented:
> Sorry for the delay - I just got back from parental leave. I'd have fixed this myself, but the comments about the defaults left me confused... What are the default values the device has when freshly reset and paired? The ones in the JSON file or the ones in the comments behind them?
> 
> Other than that I have a few minor remarks:

I have updated the default values to the one from the device firmware. The others were from the manual, which i have removed.

AlCalzone reviewed (approved):
I've removed the templates again, we're trying to get away from the super-generic ones that just specify a range.
